EngineLayer/Proteomics/CompactPeptideBase.cs

```diff
@@ -181,6 +181,40 @@ protected IEnumerable<double> ComputeFollowingFragmentMasses(PeptideWithSetModif
             } while ((oneBasedIndexToLookAt > 1 && direction == -1) || (oneBasedIndexToLookAt < yyy.Length && direction == 1));
         }
 
+        public void CropTerminalMasses(TerminusType terminusType)
```

Comment:
Sure. The method won't do anything to CompactPeptide objects if we left it where it is, but I only have to add a single "as CompactPeptideWithModifiedMass" to move it there, which isn't so bad.